MAINTAINERS.md

```diff
@@ -1,12 +0,0 @@
-# The Tinkerbell Maintainers
-
-This file lists the maintainers of the overall Tinkerbell project. The responsibilities of repository maintainers are listed in the [OWNERS.md](OWNERS.md) file.
```

Comment:
With MAINTAINERS.md being removed, how do you the relevance of https://github.com/tinkerbell/org/blob/main/OWNERS.md#composition-and-balance on admins? Should OWNERS.md (or ADMINS.md) be revised for consistency?